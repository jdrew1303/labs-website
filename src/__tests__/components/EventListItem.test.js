import React from 'react'
import EventListItem from '../../components/events/event-list-item'

const event = { "event_name": "Understanding How Collaboration Increases Productivity", }

const setEvents = () => { return; }

it('should render event name as  correctly', () => {
    const tree = (
      <EventListItem event={event} />
    )
    expect(tree).toMatchSnapshot()
});