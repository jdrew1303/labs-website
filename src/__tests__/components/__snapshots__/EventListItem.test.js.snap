exports[`test should render event name ucorrectly 1`] = `
<Unknown
  event={
    Object {
      "event_name": "Understanding How Collaboration Increases Productivity",
    }
  } />
`;
